app/models/manageiq/providers/physical_infra_manager.rb

```diff
@@ -16,5 +20,17 @@ def self.description
     def validate_authentication_status
       {:available => true, :message => nil}
     end
+
+    def count_physical_servers_with_host
+      physical_servers.inject(0) { |t, physical_server| physical_server.host.nil? ? t : t + 1 }
+    end
+
+    def total_hosts; count_physical_servers_with_host; end
```

Comment:
Implemented this as 

``alias :total_hosts :count_physical_servers_with_host``